gdb-xtensa-esp32-elf/meta.yaml

```diff
@@ -45,17 +43,20 @@ requirements:
     - libiconv {{ libiconv }}
     - mpfr {{ mpfr }}
     - gmp {{ gmp }}
-    - expat {{ expat }} # [linux]
+    - expat {{ expat }}
```

Comment:
I think we'll want to use the macOS system one, so this would be only Linux.

Reply:
ahhh, didn't realize that's how it worked, reverting